### PR TITLE
Fix cameras not reading data as bytes

### PR DESCRIPTION
**Pull request by bdraco**

### CODE REVIEW

File: src/synology_dsm/api/__init__.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+
+if TYPE_CHECKING:
+    from synology_dsm import SynologyDSM
+
 
 class SynoBaseApi:
     """Base api class."""
 
-    def __init__(self, dsm):
+    def __init__(self, dsm: "SynologyDSM") -> None:
```
Comment (mib1185) [inline src/synology_dsm/api/__init__.py:12]: uhhh ... very nice 👍

**Review by mib1185 — APPROVED** [review]
Thanks @bdraco

File: src/synology_dsm/api/__init__.py (hunk 1 above)
Comment (bdraco) [reply src/synology_dsm/api/__init__.py:12]: I added it because I was doing some right clicking and following the code, and it made it a lot easier

Comment (bdraco) [reply src/synology_dsm/api/__init__.py:12]: It might be better to make it a relative import in the future, but probably fine either way for now

**Pull request merged**